src/migrations/0.21-0.22.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path')
+const fs = require('fs')
+
+const Cache = require('orbit-db-cache')
+
+const Logger = require('logplease')
+const logger = Logger.create('orbit-db')
+Logger.setLogLevel('ERROR')
+
+async function migrate (OrbitDB, options, dbAddress) {
+  let oldCache = OrbitDB.caches[options.directory]; let oldStore
```

Comment:
Any reason not to put `let oldStore` on its own line?

Reply:
ha, actually `standard --fix` changed that from a comma to a semicolon and left it on the same line

Reply:
I'll move it down

Reply:
Oh sneaky `standard`